.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
```

Comment:
Can you double check these settings, as in `tsconfig` it has a target of `ES5` but the `env` here is allowing `es2021`

Reply:
Addressed by adding ES5 plugin